### ISSUE TITLE
Cluster bring up instructions need to include running cluster/get-kube-binaries.sh

### ISSUE THREAD

**Issue by azimbabu** [GoogleCloudPlatform/kubernetes-bigquery-python#17]
I have followed the instructions : https://cloud.google.com/solutions/real-time/kubernetes-redis-bigquery up to "Start up your cluster" section .

When i try kubernetes/cluster/kube-up.sh i get this error : 
```sh
$ kubernetes/cluster/kube-up.sh
... Starting cluster in us-central1-b using provider gce
... calling verify-prereqs

All components are up to date.

All components are up to date.

All components are up to date.
... calling kube-up
Your active configuration is: [<MY-PROJECT-ID>]

Project: <MY-PROJECT-ID>
Zone: us-central1-b
!!! Cannot find kubernetes-server-linux-amd64.tar.gz
```
I've downloaded latest kubernetes from https://dl.k8s.io/v1.5.1/kubernetes.tar.gz and my OS is Mac OSX El Capitan 10.11.16 .

Any help is greatly appreciated. Thanks!

**Comment by jeffmendoza** [comment]
I just checked the release you pointed to, and under `server/README` I ses:

```
Server binary tarballs are no longer included in the Kubernetes final tarball.

Run cluster/get-kube-binaries.sh to download client and server binaries.
```

Can you try that and re-run kube-up?

I believe kube-up is moving towards deprecation, we will need to update this sample, or just point to the official docs.

**Comment by azimbabu** [comment]
Thanks for the help @jeffmendoza . Running cluster/get-kube-binaries.sh downloaded server binary and re-running kube-up worked. 👍

**Comment by jeffmendoza** [comment]
Thanks for checking that. I edited the title and will leave this open for the instructions to be fixed

**Comment by jeffmendoza** [comment]
Closed with #21 switch instructions to GKE
